tools/test.py

```diff
@@ -1565,6 +1549,25 @@ def PrintCrashed(code):
   else:
     return "CRASHED (Signal: %d)" % -code
 
+# these suites represent special cases that should not be run as part of the
+# default JavaScript test-run, e.g., internet/ requires a network connection,
+# addons/ requires compilation.
+IGNORED_SUITES = [
+  'addons',
+  'addons-napi',
+  'gc',
+  'internet',
+  'pummel',
+  'test-known-issues',
+  'timers'
+]
+
+def DefaultSuites(test_root):
+  built_in_tests = []
+  for testdir in os.listdir(test_root):
+    if isdir(join(test_root, testdir)) and (testdir not in IGNORED_SUITES):
```

Comment:
The paren are not necessary around the `not in` expression.

Reply:
> The paren are not necessary around the not in expression.

Also a quick look at the file shows a tendency not to use them in multi termed contininals:
https://github.com/nodejs/node/blob/bf6c7eaeacf8c1ceee5c80861d10cecfe656e36b/tools/test.py#L652

<sub> I miss python </sub>